src/data/markdown/translated-guides/en/03 Using k6 browser/02 Running browser tests.md

```diff
@@ -179,30 +223,40 @@ To run a browser-level and protocol-level test concurrently, you can use [scenar
 <CodeGroup labels={["script.js"]} lineNumbers={[true]}>
 
 ```javascript
-import { chromium } from 'k6/experimental/browser';
+import { browser } from 'k6/experimental/browser';
 import { check } from 'k6';
 import http from 'k6/http';
 
 export const options = {
   scenarios: {
-    browser: {
+    testk6: {
```

Comment:
change this back to `browser`

Reply:
The last time I tried, it didn't work because we imported the module as `browser`. I'll give it another try.

Reply:
Resolved in c13d101213b41ded93d5ae5d772b5ecbca3fd9d2